class-list-table.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * List table for displaying the list of sites.
+ *
+ * @package Pantheon_Decoupled
+ */
+
+use WP_List_Table;
```

Comment:
As I'm sure you're aware, there is a lot of redundant code here. I think it should be possible to instead extend/use decoupled-preview's `List_Table` class since the base of what we want here is very similar. To start, rather than using `WP_List_Table` try:

`use Pantheon\DecoupledPreview\List_Table;`

Which should allow us to actually use the existing list table.